extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -13,151 +13,26 @@
-import {getSocialConfig} from './amp-social-share-config';
-import {isExperimentOn} from '../../../src/experiments';
-import {toggle} from '../../../src/style';
-import {user, userAssert} from '../../../src/log';
 
 /** @const {string} */
 const TAG = 'amp-social-share';
-
-/**
- * @private
- * @param {string} type
- * @param {!../../../src/service/viewer-interface.ViewerInterface} viewer
- * @param {!../../../src/service/platform-impl.Platform} platform
- * @return {!JsonObject|undefined}
- */
-const getTypeConfigOrUndefined = (type, viewer, platform) => {
-  if (type === 'system') {
-    // navigator.share unavailable
-    if (!systemShareSupported(viewer, platform)) {
-      return;
-    }
-  } else {
-    // system share wants to be unique
-    const systemOnly =
-      systemShareSupported(viewer, platform) &&
-      !!window.document.querySelector(
-        'amp-social-share[type=system][data-mode=replace]'
-      );
-    if (systemOnly) {
-      return;
-    }
-  }
-  const typeConfig = getSocialConfig(type) || dict();
-  if (typeConfig['obsolete']) {
-    user().warn(TAG, `Skipping obsolete share button ${type}`);
-    return;
-  }
-  return typeConfig;
-};
-
-/**
- * @private
- * @param {!../../../src/service/viewer-interface.ViewerInterface} viewer
- * @param {!../../../src/service/platform-impl.Platform} platform
- * @return {boolean}
- */
-const systemShareSupported = (viewer, platform) => {
-  // Chrome exports navigator.share in WebView but does not implement it.
-  // See https://bugs.chromium.org/p/chromium/issues/detail?id=765923
-  const isChromeWebview = viewer.isWebviewEmbedded() && platform.isChrome();
-
-  return 'share' in navigator && !isChromeWebview;
-};
+const DEFAULT_BINDINGS = ['TITLE', 'CANONICAL_URL'];
 
 class AmpSocialShare extends PreactBaseElement {
   /** @override */
   init() {
-    const viewer = Services.viewerForDoc(this.element);
-    const platform = Services.platformFor(window);
-    const type = userAssert(
-      this.element.getAttribute('type'),
-      'The type attribute is required. %s',
-      this.element
-    );
-    userAssert(
-      !/\s/.test(type),
-      'Space characters are not allowed in type attribute value. %s',
-      this.element
-    );
-    const typeConfig = getTypeConfigOrUndefined(type, viewer, platform);
-    // Hide/ignore component if typeConfig is undefined
-    if (!typeConfig) {
-      toggle(this.element, false);
-      user().warn(TAG, `Skipping obsolete share button ${type}`);
-      return;
-    }
-    this.renderWithHrefAndTarget_(typeConfig, platform);
-    const responsive =
-      this.element.getAttribute('layout') === Layout.RESPONSIVE && '100%';
-    return dict({
-      'width': responsive || this.element.getAttribute('width'),
-      'height': responsive || this.element.getAttribute('height'),
-      'href': null,
-      'target': null,
-    });
-  }
-
-  /** @override */
-  isLayoutSupported() {
```

Comment:
I believe we still need this experiment assertion here?

Reply:
Thanks will update!